cdc_iterator.go

```diff
@@ -197,33 +197,39 @@ func (c *cdcIterator) buildRecord(ctx context.Context, e rowEvent) (opencdc.Reco
 	// payload really is just an alias, but makes buildRecord easier to understand.
 	payload = payloadBefore
 
-	keyCol := c.config.tableSortCols[tableName]
+	keyCol := c.config.primaryKeys[tableName]
```

Comment:
Since this variable now represents a map, I think we should update its name. Suggestion:

```suggestion
	keyCols := c.config.primaryKeys[tableName]
```